ops/storage-ddls/cassandra-setup.cql

```diff
@@ -68,6 +70,7 @@ DROP TABLE IF EXISTS sitesettings;
 CREATE TABLE sitesettings(
     sitename text,
     geofence list<double>,
+    places list<place>,
```

Comment:
For usage example, see the PR that is linked above: https://github.com/CatalystCode/project-fortis-services/pull/91
In essence, whenever we fetch all the places for a pre-computed geofence, we'll use these saved places instead of calling the featureService as a large bounding box call will be expensive. @jcjimenez has more background since he suggested this, so you may want to chat with him also.